Lib/test/test_uu.py

```diff
@@ -251,10 +271,7 @@ def test_decodetwice(self):
             self._kill(f)
 
 def test_main():
-    support.run_unittest(UUTest,
-                              UUStdIOTest,
-                              UUFileTest,
-                              )
+    support.run_unittest(UUTest, UUStdIOTest, UUFileTest)
```

Comment:
This seems like an insignificant change. Wouldn’t it be simple to use “unittest.main” instead?